README.md

```diff
@@ -13,13 +15,23 @@ The code has been copied and adapted from https://gist.github.com/tstachl/626424
 TBD
 
 ## Usage
-Example:
+Examples:
 
 ```
 require 'rack/content-type-default`
 config.middleware.use Rack::ContentTypeDefault, :post, 'application/x-www-form-urlencoded'
 ```
 
+```
+require 'rack/content-type-default`
+config.middleware.use Rack::ContentTypeDefault, [:get, :post], 'application/xml', '/authenticate.xml'
+```
+
+```
+require 'rack/content-type-default`
+config.middleware.use Rack::ContentTypeDefault, :post, 'application/json', ['/authenticate.json', '/show'], true
```

Comment:
This `true`/false option isn't described in the Readme. Can you please elaborate on it above in the to-be-numbered list?